karspexet/templates/base.html

```diff
@@ -0,0 +1,91 @@
+{% load menu_tags cms_tags sekizai_tags staticfiles%}
```

Comment:
After some experimenting I found that Django templets do not like the `<!DOCTYPE html>` tag. And the styling/padding is changed when it is used...